### DISCUSSION TITLE
Synology Client

### DISCUSSION THREAD

**Discussion by stefanocutelle** [gravitl/netmaker#1441] [Q&A] [answered]
good morning Mate, i really like your project, i think a very useful feature would be to have a native client for synology, maybe through the community portal, synocommunity.com. what do you think ? is it feasible in a short time?

**Comment by afeiszli** [comment, marked as answer]
We dont have an "official" synology client, but there is a guide to run on synology: https://servicemax.com.au/tips/wireguard-and-netmaker-on-synology/
